File: src/Ejercicio13/Ejercicio13.java
package Ejercicio13;

public class Ejercicio13 {



    public static void main(String[] args) {
        Anime anime1 = new Anime("Bleach", "Shonen", 523);
        Anime anime2 = new Anime("Dragonball", "Shonen", 1000);


        System.out.println(anime1);
        System.out.println(anime2);
    }




    public static class Anime {

        private String _nombre;
        private String _genero;
        private int _episodios;

        public Anime(String nombre, String genero, int episodios) {

            _nombre = nombre;
            _genero = genero;
            _episodios = episodios;
        }


        public String toString(){
            return "Anime: " + _nombre + " es del género " + _genero + " y cuenta con " + _episodios + " episodios.";
        }
    }
}

File: src/Ejercicio13/Ejercicio13.kt
package Ejercicio13

fun main(){
    val anime1 = Anime("Bleach", "Shonen", 523)
    val anime2 = Anime("Dragonball", "Shonen", 1000)

    println(anime1)
    println(anime2)
}


class Anime(val nombre: String, val genero: String, val episodios: Int){


    override fun toString(): String {
        return "Anime: $nombre es del género $genero y cuenta con $episodios episodios."
    }
}